### DISCUSSION TITLE
Quick Scan

### DISCUSSION THREAD

**Discussion by fazim-sec** [hahwul/dalfox#367] [Q&A] [answered]
Hey is there a quick scan option in dalfox, that is a scan quickly checking for xss or mysql error then move to next url in a file.

**Comment by hahwul** [comment, marked as answer]
Hi @fazim-pentester 
I think check sql error will be included on its own because it's in the form of passive scan(BAV). 
Are you talking about a faster type of scan than now?

I'm going to speed up in the next version of dalfox, but if quick-scan is a really simple check XSS (the check reflection of special characters), I think it's a good idea to make a separate type.

**Reply by fazim-sec** [reply]
Hey @hahwul 
Yes quick scan for really simple checks is what i want, because of long list of url's it takes more time to check. It would be great if a separate type for quickscan is available for dalfox therefore speeding up this processs.
